app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -98,7 +98,7 @@ ImportProcessClass to_import_process_class(const G4VProcess& process)
             {"ionIoni",        ImportProcessClass::ion_ioni},
             {"msc",            ImportProcessClass::msc},
             {"hIoni",          ImportProcessClass::h_ioni},
-            {"hBrems",         ImportProcessClass::h_brems},
+            {"hBrem",          ImportProcessClass::h_brem},
```

Comment:
Yes. It is `hBrems` in Geant4:

`G4hBremsstrahlung.hh`
```c++
  explicit G4hBremsstrahlung(const G4String& processName = "hBrems");
```

Reply:
Thanks!

Reply:
Same for `muBrems`.

Reply:
@vrpascuzzi it looks like your PR still changes the enum, even though it was originally "correct" (written as intended for consistency with the G4 class names). I see your note that you changed these for "consistency", but I don't understand with what?

Reply:
There is a mix of `*brem` and `*brems` in these import classes, e.g., [`h_brems`](https://github.com/celeritas-project/celeritas/blob/master/app/geant-exporter/ImportProcessConverter.cc#L101), [`e_brem`](https://github.com/celeritas-project/celeritas/blob/master/app/geant-exporter/ImportProcessConverter.cc#L105), [`mu_brems`](https://github.com/celeritas-project/celeritas/blob/master/app/geant-exporter/ImportProcessConverter.cc#L112).

Looking again at @stognini's reference and related ones, I understand now the reason for doing it this way. However, I think within celeritas it's important we have a common 'value' that maps to whatever mindset G4 had when aliasing their processes.

What I'm suggesting is that, in `ImportProcess.hh`, we use `h_brems`, `e_brems` and `mu_brems` rather than [...], `e_brem` [...]. It's OK to follow G4 to some extent but also must keep our sanity (and spare that of future users) when we come across such apparent illogical conventions.

So we can either (a) revert these changes or (b) make things right. :)

Reply:
Aha, I see. Well, you know I'm a stickler for consistency and not often a fan of "this is how Geant does it". So I'm up for changing! Since I thought the natural abbreviation (at least how I speak and perceive others shortcuts as well) was "brems" I'm making a poll 😉 

https://celeritasproject.slack.com/archives/CSLM7G17V/p1622288259002800

Reply:
And now that we're in the consistency can of worms... Geant says `Rayl` and `compt` as if Rayleigh is a person and Compton isn't 🙄